debian/rules

```diff
@@ -97,6 +97,11 @@ override_dh_auto_install:
 	# Remove j2lint comments / linter configuration which would insert additional new-lines
 	find $(DIR)/$(VYOS_DATA_DIR) -name "*.j2" -type f | xargs sed -i -e '/^{#.*#}/d'
 
+ifeq ($(DEB_HOST_ARCH),arm64)
+	# T7484: On ARM64, remove the console ttyS0 section from default config, breaks if there is no such device
+	find $(DIR)/$(VYOS_DATA_DIR) -name "config.boot.default" -type f | xargs sed -i -e '/console {/,/^    }/d'
```

Comment:
This change seems so hacky
Any other ideas?

Reply:
I agree, although I did the same as the existing inline sed replacement just a line above that removes the j2lint comments.

Other ideas:

1. Add `config.boot.default.arm64` and copy that specifically on arm64,  otherwise `config.boot.default`. Downside is that you need to maintain two copies.

2. Just remove the whole `console` section from the _default_ config, is that possible? As I don't fully understand why it is there by default and I also don't understand the relation of the GRUB boot options and the setting in the config and how this all works. If someone can explain I could maybe also think of other options.